Solutions/CiscoSEG/Hunting Queries/CiscoSEGUsersReceivedSpam.yaml

```diff
@@ -0,0 +1,27 @@
+id: 7895ffa5-cd61-43cf-89e5-9630e79685fd
+name: Cisco SEG - Top users receiving spam mails
+description: |
+  'Query searches for top users receiving spam mails.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: CiscoSEG
+    dataTypes:
+      - CiscoSEGEvent
+tactics:
+  - InitialAccess
+relevantTechniques:
+  - T1566
+query: |
+  CiscoSEGEvent
+  | where TimeGenerated > ago(24h)
+  | where MailDirection =~ 'Incoming'
+  | where SimplifiedDeviceAction =~ 'QUARANTINED'
+  | extend act_det = extract(@'ESAFinalActionDetails":"(.*?)"', 1, tostring(AdditionalFields))
+  | where act_det has 'To SPAM'
+  | summarize count by DstUserName

```

Comment:
Since we are only looking for top users receiving spam mails vs all users; wondering if after summarize do we need something like 
| where count_ > ( some threshold value)

Reply:
here we are looking for all users receiving spam mails and aggregation is used to sort them by amount of spam mails. Thus suggested condition is optional here.